applications/basic_networking_ping/cpp/CMakeLists.txt

```diff
@@ -31,11 +31,11 @@ target_link_libraries(basic_networking_ping
 
 # Copy config file
 add_custom_target(basic_networking_ping_rx_yaml
-  COMMAND ${CMAKE_COMMAND} -E copy "${CMAKE_CURRENT_SOURCE_DIR}/../basic_networking_ping_rx.yaml" ${CMAKE_CURRENT_BINARY_DIR}
+  COMMAND ${CMAKE_COMMAND} -E copy "${CMAKE_CURRENT_SOURCE_DIR}/../basic_networking_ping_rx.yaml" ${CMAKE_BINARY_DIR}
```

Comment:
Is there any reason that `CMAKE_BINARY_DIR` makes sense here?

Many apps in Holohub is still using `CMAKE_CURRENT_BINARY_DIR`. Should we revisit other CMakeLists.txt files?

Reply:
This change allows the user to launch the application using `./devcontainer build_and_run` and specify the config file as put in the README.

Reply:
@mocsharp Could we please instead update the default `run` command in the app [`metadata.json`](https://github.com/nvidia-holoscan/holohub/blob/main/applications/basic_networking_ping/cpp/metadata.json#L42) to launch relative to `holohub_app_bin`, aka `CMAKE_CURRENT_BINARY_DIR`? I agreed with @gigony that `CMAKE_CURRENT_BINARY_DIR` would be better aligned with the Python app version along with other HoloHub apps.

Reply:
Thank you @tbirdso for the tip. I have updated and tested the app.